### ISSUE TITLE
how to install gcc-7.3 or gcc-7.2?

### ISSUE THREAD

**Issue by hmaarrfk** [Archiconda/build-tools#2]
@jjhelmus I've managed to make the first package, toolchain,

but it seems I need a compatible version of `libgcc-ng`. I can't find the feedstock for that one.

It looks from the notes that you cross compiled that one.

Could you help me get it to the version I have installed in the docker container??
The docker container uses gcc 7.4.
I don't really know how to install gcc 7.3 on aarch64

You can see the results of this build process:
https://app.shippable.com/github/hmaarrfk/make-feedstock/runs/4/1/console

I just got the rights for Archiconda to use aarch instances on shippable.

**Comment by hmaarrfk** [comment]
It probably has something to do with this:
https://github.com/jjhelmus/conda4arm_bootstrap
But this seems to be use cross compilers. Something I have no experience with.

**Comment by hmaarrfk** [comment]
I think my previous thinking was wrong. @jjhelmus had compiled compilers for us and put them in his channel

https://anaconda.org/jjhelmus/repo?label=aarch64_bootstrap

Instead of using the toolchain compilers, I think I am supposed to use the `gcc` compilers.

**Comment by jjhelmus** [comment]
Yes, there are GCC 7.3.0 compilers packages in the [jjhelmus/label/aarch64_bootstrap](https://anaconda.org/jjhelmus/repo?label=aarch64_bootstrap) channel that can be used.  Keep in mind that I created these bootstrap packages on a machine running Ubuntu 18.04 so the compilers require glibc 2.27 or newer.  These compilers produce binaries that target glibc 2.17 and therefore the artifacts should work with CentOS 7 and newer distribution (CentOS 6 does not support aarch64).  

I'm working on rebuilding the compilers in a CentOS 7 docker container to produce compiler packages that are compatible with broader array of glibc versions.
